### PR TITLE
Change move show complete map button under the viewcube

### LINKED ISSUE #642: Reposition Show Complete Map-Button
# Feature request

## Description
LegendPanel now overlaps with the Show Complete Map-Button

Possibilities:
- [ ] Move LegendPanel up a bit and place the button down below
- [x] Move the Button under the viewcube
- [ ] Move the Button to the filepanel

Additionally, we could think about an icon, that indicates this purpose. We use a bullseye to represent focusing. We could use the same icon and cross it with a line or something to indicate unfocussing.

### PR DESCRIPTION
**Pull request by NearW**
# Move the show complete map button under the viewCube

closes #642 

## Description

- Moved the show complete map button under the viewcube

### Before
image

### After
image

## Definition of Done

A task/pull request will not be considered to be complete until all these items can be checked off.

- [x] **All** requirements mentioned in the issue are implemented
- [x] Does match the Code of Conduct and the Contribution file
- [x] Task has its own **GitHub issue** (something it is solving)
  - [x] Issue number is **included in the commit messages** for traceability
- [x] **Update the README.md** with any changes/additions made
- [x] **Update the CHANGELOG.md** with any changes/additions made
- [x] **Enough test coverage to ensure that uncovered changes do not break functionality**
- [x] **All tests pass**
- [x] **Descriptive pull request text**, answering:
  - What problem/issue are you fixing?
  - What does this PR implement and how?
- [x] **Assign your PR to someone** for a code review
  - This person _will be contacted **first**_ if a bug is introduced into `master`
- [x] **Manual testing** did not fail

### CODE REVIEW

File: visualization/app/codeCharta/ui/codeMap/codeMap.component.scss [hunk 1]
```diff
@@ -9,4 +9,25 @@ code-map-component {
 			right: 350px;
 		}
 	}
+
+	.action-buttons {
```
Comment (alschmut) [inline visualization/app/codeCharta/ui/codeMap/codeMap.component.scss:13]: When you are already working on that button class:
- Do you want to simplify the CSS? Can we maybe get almost rid of it by using the `small-action-button` class?
- Shouldn't we move that styling to the `unfocusButton.component.scss`

Comment (NearW) [reply visualization/app/codeCharta/ui/codeMap/codeMap.component.scss:13]: I agree with the Unfocus description.

Right now there is only one action-button, so the class `action-buttons` is not really meaning-ful. Instead let's move the css to the unfocus-button-component as suggested.

**Comment by NearW** [comment]
image

File: visualization/app/codeCharta/ui/codeMap/codeMap.component.scss (hunk 1 above)
Comment (NearW) [reply visualization/app/codeCharta/ui/codeMap/codeMap.component.scss:13]: Done

Comment (NearW) [reply visualization/app/codeCharta/ui/codeMap/codeMap.component.scss]: what we can do as well is, if you right-click the focused building, that you can instead unfocus it.

**Comment by NearW** [comment]
@alschmut Let's keep this PR open for now. We should discuss if the unfocus button should be moved to the nodeContextMenu in the planning on friday.

**Comment by alschmut** [comment]
@NearW yea sure. I never really liked the outstanding button, but I also think we need it as some users might not even know there is a contextMenu. For another feature we could enable the nodeContextMenu when only right clicking on the canvas background.

**Comment by NearW** [comment]
@alschmut True. We should provide some sort of tutorial with the basic features of codecharta for that in our documentation.

The NodeContextMenu belongs to a node. Right-clicking on the canvas has no dedicated node.

**Comment by alschmut** [comment]
@NearW I know, but for some actions like infocusing it might make sense to provide those in a separate contextMenu

**Comment by NearW** [comment]
Mhh, idk how we handle moving the map then. Holding right-click moves the map.
I think that the focus is something node related. You decide to focus a node. Therefore I as a user would assume, that in order to unfocus it, I have to reopen the nodeContextMenu and select unfocus.

**Pull request merged**